internal/command/jsonformat/plan.go

```diff
@@ -463,20 +472,28 @@ func resourceChangeHeader(change jsonplan.ResourceChange) string {
 	return fmt.Sprintf("%s \"%s\" \"%s\"", mode, change.Type, change.Name)
 }
 
-func actionDescription(action plans.Action) string {
+func actionDescription(action plans.Action, importing bool) string {
 	switch action {
 	case plans.Create:
 		return "  [green]+[reset] create"
 	case plans.Delete:
 		return "  [red]-[reset] destroy"
```

Comment:
Is there going to be special rendering for imports that end with with destroy or replacement actions?

Reply:
Yes, I'm using this PR as a base to finish off the implementation here and we will have dedicated rendering for imports that do some thing else (eg. get destroyed/updated/replaced/moved).